### ISSUE TITLE
Recreate with flutter

### ISSUE THREAD

**Issue by hiranthaR** [scorelab/SCoRe-Mobile-App#5]
We know flutter is now very popular cross-platfom framework these days.I suggest, re build this project with flutter.

**Comment by dextroxd** [comment]
Please assign me this issue. @hiranthaR

**Comment by dilumdesilva** [comment]
I strongly agree with @hiranthaR since Flutter is the latest google open-source mobile framework and technology I would like to work on the Flutter project if we are migrating to flutter.
